apis/mattermost/v1alpha1/clusterinstallation_sizes.go

```diff
@@ -96,17 +101,19 @@ var cloudSize10 = ClusterInstallationSize{
 
 // CloudSize100String represents estimated Mattermost Cloud installation sizing for 100 users.
 const CloudSize100String = "cloud100users"
+var cpuInstallationRequests100u = getEnv("MM100_CPU_REQUESTS", "50m")
+var memoryInstallationRequests100u = getEnv("MM100_MEM_REQUESTS", "256Mi")
```

Comment:
Same here, although it doesn't remind me the licensing, just to follow the pattern